.github/workflows/ci.yml

```diff
@@ -0,0 +1,110 @@
+name: CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+env:
+  SERVER_ASSET: trypurescript-server
+  CLIENT_ASSET: trypurescript-client
+
+jobs:
+  build_server:
+    name: Build server
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: haskell/actions/setup@v1
+        with:
+          enable-stack: true
```

Comment:
Should we pin the stack version here at all? See for example, the [PureScript ci.yml file](https://github.com/purescript/purescript/blob/master/.github/workflows/ci.yml#L37-L39).

Also, should we set the [`stack-no-global` flag to `true`](https://github.com/haskell/actions/tree/main/setup#inputs)?